src/client/components/CreateSeason.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+
+import { TitleForm } from './seasonform/TitleForm';
+import { CompetitorForm } from './seasonform/CompetitorForm';
+import { TeamForm } from './seasonform/TeamForm';
+import { ParticipantForm } from './seasonform/ParticipantForm';
+import { MappingForm } from './seasonform/MappingForm';
+
+export const CreateSeason = () => {
+
+    const competitionId = 1;
+
+    const [form, setForm] = useState({
+        competitionId: competitionId,
+        title: "",
+        participants: [],
+        teams: [],
+        mapping: [],
+    })
+    const [formStep, setFormStep] = useState(1)
+
+    useEffect(() => {
+        if(formStep === 6) postForm()
```

Comment:
is there maybe a better more descriptive way of representing formsteps instead of numbers

Reply:
formStep itself is best as an int, as this will eventually be used in styling variables to translate an element using formStep as a multiple. It also allows us to better catch button presses that are out of order. However you are right, the conditionals could be more descriptive... 6 means nothing here unless you already know there's 5 steps to the form